cpplint/cpplint.py

```diff
@@ -4291,6 +4292,15 @@ def GetLineWidth(line):
       if unicodedata.east_asian_width(uc) in ('W', 'F'):
         width += 2
       elif not unicodedata.combining(uc):
+        # Issue 337        
+        if (sys.version_info.major, sys.version_info.minor) == (2, 7) :
```

Comment:
I suggest writing this as `if sys.version_info <= (2,):` as this is not == 2.7 specific.  technically `<= (3,2)` makes sense but we're never going to support the long obsolete early 3.0-3.2 interpreters.

Reply:
Thanks!